### PR TITLE
Fix title classes

### PR DESCRIPTION
**Pull request by pieer**
Remove unnecessary class when subtitle + is-title=[x]
+ fix bug on c-title

### CODE REVIEW

File: frontend/views/containers/contributions/GroupPledgesGraph.vue [hunk 1]
```diff
@@ -4,7 +4,7 @@
     :slices='mainSlices'
     :inner-slices='innerSlices'
   )
-    i18n.is-title-1.c-title(tag='h3' :args='LTags("span")') {span_}Group{_span} goal
+    i18n.has-text-1.is-title-3.c-title(tag='h3' :args='LTags("span")') {span_}Group{_span} goal
```
Comment (sandrina-p) [inline frontend/views/containers/contributions/GroupPledgesGraph.vue]: We don't need `.is-title-x` here, this doesn't need to look like a title, is just a "normal" test with `.has-text-1` color. 

image

Comment (pieer) [reply frontend/views/containers/contributions/GroupPledgesGraph.vue]: Mmm we actually stipulate in the guide that we need to tell the class all the time isn'
t it?

Comment (sandrina-p) [reply frontend/views/containers/contributions/GroupPledgesGraph.vue]: We need to add the class if the UI matches. In this case, the UI doesn't look like a title, so there's no need for it:
image

Comment (pieer) [reply frontend/views/containers/contributions/GroupPledgesGraph.vue]: Confusing.. anyway, there is no class is-text-[x] for 16px or 1rem

Comment (pieer) [reply frontend/views/containers/contributions/GroupPledgesGraph.vue]: A 16px h3 title with no class

File: frontend/views/components/modal/ModalTemplate.vue [hunk 2]
```diff
@@ -14,10 +14,10 @@
       .c-modal-content(ref='card' v-if='modalIsActive')
         header.c-modal-header(
           :class='{ "has-subtitle": $scopedSlots.subtitle }'
-          v-if='$scopedSlots.title || $scopedSlots.subTitle'
+          v-if='$scopedSlots.title || $scopedSlots.is-subtitle'
```
Comment (sandrina-p) [inline frontend/views/components/modal/ModalTemplate.vue]: I think this was a wrong find-replace. it should be subtitle as in the line 20

File: frontend/views/containers/contributions/GroupPledgesGraph.vue (hunk 1 above)
Comment (sandrina-p) [reply frontend/views/containers/contributions/GroupPledgesGraph.vue]: Okay, so another approach, that makes more sense:
Leave this "Group Goal" as a normal text (without heading) as we usually do.
Add the title of this graphic/summary top of the component and hide it using `.sr-only`

```pug
i18n.sr-only(tag='h3') Group Pledging Status
```

That way we serve both worlds visually and semantically.

**Review by sandrina-p — CHANGES_REQUESTED** [review]
There was a wrong replacement on the code with `subtitle`. check the comment I left.

File: frontend/views/components/modal/ModalTemplate.vue (hunk 2 above)
Comment (pieer) [reply frontend/views/components/modal/ModalTemplate.vue]: Fixed. Thanks

File: frontend/views/components/modal/ModalTemplate.vue [hunk 3]
```diff
@@ -14,10 +14,10 @@
       .c-modal-content(ref='card' v-if='modalIsActive')
         header.c-modal-header(
           :class='{ "has-subtitle": $scopedSlots.subtitle }'
-          v-if='$scopedSlots.title || $scopedSlots.subTitle'
+          v-if='$scopedSlots.title || $scopedSlots.issubtitle'
```
Comment (sandrina-p) [inline frontend/views/components/modal/ModalTemplate.vue]: @pieer this should be `$scopedSlots.subtitle`

**Review by sandrina-p — CHANGES_REQUESTED** [review]
you fixed the typo with another typo 😆

File: frontend/views/components/modal/ModalTemplate.vue (hunk 3 above)
Comment (pieer) [reply frontend/views/components/modal/ModalTemplate.vue]: Of course... I'm tired today...

**Review by sandrina-p — APPROVED** [review]
okay, it seems all fine now! I'm merging this since it's something small

**Pull request merged**